modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -19503,6 +19503,7 @@ to-is-required=To is required.
 to-manage-priorities,-drag-the-rule-to-the-desired-position=To manage priorities, drag the rule to the desired position.
 to-navigate=to navigate
 to-save-this-form-all-required-fields-of-the-selected-object-need-to-be-mapped=To save this form, all required fields of the selected object need to be mapped.
+to-see-the-stack-trace-click-on-this-notification=To see the stack trace click on this notification.
```

Comment:
Reword this to be consistent with `click-on-this-notification-to-see-the-list-of-conflicts-that-need-to-be-manually-resolved`